float64 takeoff_height
---
---
float64 current_height